consensus/hotstuff/committees/consensus_committee.go

```diff
@@ -62,11 +65,8 @@ func newStaticEpochInfo(epoch protocol.Epoch) (*staticEpochInfo, error) {
 	}
 
 	totalWeight := initialCommittee.TotalWeight()
-	epochInfo := &staticEpochInfo{
-		firstView:            firstView,
-		finalView:            finalView,
-		randomSource:         randomSource,
-		leaders:              leaders,
+	epochInfo := &epochInfo{
```

Comment:
would suggest avoiding to shadow the struct declaration here. Using a very short acronym here does not impact readability here, because the struct initialization makes the type direly apparent

```suggestion
	ei := &epochInfo{
```